WcaOnRails/app/views/admin/new_results.html.erb

```diff
@@ -1,9 +1,12 @@
 <% provide(:title, "Upload results") %>
 <%= render layout: 'competitions/nav' do %>
   <h1><%= yield(:title) %></h1>
-  <P>
+  <p>
     Import the results JSON for this competition.
-    The server will check for any error compared to the declared rounds, cutoff, and time limit
+    The server will check for any error compared to the declared rounds, cutoffs, and time limits.
+  </p>
+  <p>
+    When you are done checking the results, you can go ahead and post them <%= link_to "here", "#{root_url}results/admin/upload_results.php?competitionId=#{@competition.id}" %>.
```

Comment:
nit: Internally, I think it's better to use root-relative links rather than including the protocol and domain again, so instead this would be `/results/admin/upload_results.php?competitionId=#{@competition.id}`. The notable exception to this is when we send emails.